backoffice/src/wwwroot/scss/site.scss

```diff
@@ -56,3 +56,17 @@ div.form-sections {
     background-color: var(--bs-danger-bg-subtle);
     border-color: var(--bs-danger-border-subtle);
 }
+
+.progress {
+    .progress-bar {
+        border-right: 1px solid #000;
+    }
+
+    :last-child {
+        border-right: none;
+    }
+}
+
+.button-text {

```

Comment:
Hmmm, not sure about this one. I think wrap is fine in certain situations and we don't want this absolutely everywhere (esp for mobile views etc)

If it's a specific case where you absolutely want it in all cases, in scss you can wrap it in a context

.thisDiv{
   .buttonTextInHere{
     some: style;
   }
}

Reply:
Preferably, you can also put the bootstrap class for this straight on the button you don't want to wrap:

`<p class="text-nowrap">No wrap text.</p>`

Reply:
Having a play around with this and I don't think the .button-text class is necessary

Reply:
Bin it off